### PR TITLE
Sprint/2101

### PR DESCRIPTION
**Pull request by bp-spanic897**
XIONE-936 : Port TextToSpeech plug-in onto Realtek platform

Reason for change : Modified TextToSpeech code to add Realtek pipeline

Test Procedure : Build & verify voice guidance

Risks : Medium

### CODE REVIEW

File: TextToSpeech/impl/TTSSpeaker.cpp [hunk 1]
```diff
@@ -394,7 +433,11 @@ void TTSSpeaker::createPipeline() {
     }
 
     // set the TTS volume to max.
+#if defined(PLATFORM_REALTEK)
+    g_object_set(G_OBJECT(m_audioVolume), "volume", (double) (m_defaultConfig.volume() / MAX_VOLUME), NULL);
```
Comment (varumugam123) [inline TextToSpeech/impl/TTSSpeaker.cpp:437]: could you confirm if volume element expects the "volume" value between 0-1?

Comment (bp-spanic897) [reply TextToSpeech/impl/TTSSpeaker.cpp:436]: realtek also expects volume between rang 0-1
My custom build was verified by @mark.yang

File: TextToSpeech/impl/TTSSpeaker.cpp [hunk 2]
```diff
@@ -652,8 +699,8 @@ void TTSSpeaker::speakText(TTSConfiguration config, SpeechData &data) {
         m_currentSpeech = &data;
 
         g_object_set(G_OBJECT(m_source), "location", constructURL(config, data).c_str(), NULL);
-        // PCM Sink seems to be accepting volume change before PLAYING state
         g_object_set(G_OBJECT(m_audioSink), "volume", (double) (data.client->configuration()->volume() / MAX_VOLUME), NULL);
```
Comment (varumugam123) [inline TextToSpeech/impl/TTSSpeaker.cpp:702]: should this be m_audioVolume?

Comment (bp-spanic897) [reply TextToSpeech/impl/TTSSpeaker.cpp]: Corrected the same and pushed the changes

**Review by varumugam123 — APPROVED** [review]

**Review by ramasat — APPROVED** [review]

**Pull request merged**